R/app_server.R

```diff
@@ -6,7 +6,6 @@
 #' @noRd
 app_server <- function(input, output, session) {
   # Your application server logic
-  mod_markdown_example_server("markdown_example_ui_1")
-  mod_chart_example_server("chart_example_ui_1")
-  mod_scrollytell_example_server("scrollytell_example_1")
+  mod_chart_example_server("chart_example")
+  mod_scrollytell_example_server("scrollytell_example")
```

Comment:
Just removed a server that is not actually used, and renamed the golem namespaces, as `ui_1` did not make sense.

Reply:
Question! golem namespaces were copied from template (golem::add_module()), do you think it is better to edit (like remove _1 etc?)

Reply:
You can use any string as a module namespace. For our main modules, these are one-offs. In general tho, golem is designed to allow for creating composable modules, used multiple times in a nested way. That is why creating a module using `golem::add_module()` will have e.g. for module "foobar" the suggestion "foobar_1" as namespace.

Using this namespacing is not great, as "foobar_1" does not tell you anything about the context of the namespace. I made the changes to remove both the numbers, as they are not necessary for these main one-off modules, and also the "ui" in the chart example, as it was possibly confusing - why is it called ..._ui in the server module?